components/Appbar.tsx

```diff
@@ -26,7 +26,7 @@ export const Appbar = () => {
     <PaperAppbar
       style={{
         paddingTop: top,
-        backgroundColor: theme.colors.background,
+        backgroundColor: 'white',
```

Comment:
let's stick to theme colors only

Reply:
In figma, the screen background is white. I should add 'background' to theme colors as '#FFF' in order to override it.

Reply:
Then figma needs to fix the discrepancy between the defined palette and the used colors.
We'll raise this issue to UX, meanwhile let's stick to the theme defined colors. No hardcoding off other colors.

Reply:
Ok. I've switched back to `theme.colors.background`.